internal/pkg/vault/secrets.go

```diff
@@ -452,3 +452,74 @@ func (c *Client) store(subPath string, secrets map[string]string) error {
 
 	return nil
 }
+
+// GetKeys retrieves the keys at the provided sub-path. Secret Store returns an array of keys for a given path when
+// retrieving a list of keys, versus a k/v map when retrieving secrets.
+func (c *Client) GetKeys(subPath string) ([]string, error) {
+	data, err := c.getAllPaths(subPath)
+	if err != nil {
+		return nil, err
+	}
+
+	var validKeys []string
+	for _, key := range data {
+		if len(key) == strings.Index(key, "/")+1 || strings.Index(key, "/") < 1 {
```

Comment:
Would HasSuffix() work better here?

Reply:
I removed this check do to me thinking a path + subpath could be returned. This check was removing it from the list.